examples/server/webui/src/main.js

```diff
@@ -21,7 +21,7 @@ const CONFIG_DEFAULT = {
   systemMessage: 'You are a helpful assistant.',
   showTokensPerSecond: false,
   // make sure these default values are in sync with `common.h`
-  samplers: 'dkypmxt',
+  samplers: 'edkypmxt',
```

Comment:
@ngxson Note that I made this change to the WebUI and AFAIU I have to build the new `index.html` with `npm run build`. But this command is failing on my computer with the error in the previous comment.

Reply:
Don't worry I can push the built index.html to this PR.

If you want to build it on your computer, maybe you can install a version manager like [nvm](https://github.com/nvm-sh/nvm), then `nvm use 22.11.0` to use the correct version

Reply:
note: we can also have a manually run CI job that build the frontend and output an artifact, I'll add this in the future

Reply:
Ok thanks. I managed to install `22.11.0` with `nvm` and it works now 👍